### ISSUE TITLE
Relink `/lib/x86_64-linux-gnu/libkmod.so.2' with `/lib/x86_64-linux-gnu/libpthread.so.0' for IFUNC symbol `system'

### ISSUE THREAD

**Issue by mramotowski** [pmem/issues#806]
Found on 25a4fcdaa4042f6dc350b7444b17f6bd5cc3032b on Ubutnu17.10
Each time I run tests I get msg 

> Relink `/lib/x86_64-linux-gnu/libkmod.so.2' with `/lib/x86_64-linux-gnu/libpthread.so.0' for IFUNC symbol `system'

```
./RUNTESTS obj_pool -s TEST0
tools/pmemdetect/pmemdetect.static-nondebug: Relink `/lib/x86_64-linux-gnu/libkmod.so.2' with `/lib/x86_64-linux-gnu/libpthread.so.0' for IFUNC symbol `system'
tools/pmemdetect/pmemdetect.static-nondebug: Relink `/lib/x86_64-linux-gnu/libkmod.so.2' with `/lib/x86_64-linux-gnu/libpthread.so.0' for IFUNC symbol `system'
obj_pool/TEST0: SETUP (check/pmem/debug)
./obj_pool: Relink `/lib/x86_64-linux-gnu/libkmod.so.2' with `/lib/x86_64-linux-gnu/libpthread.so.0' for IFUNC symbol `system'
obj_pool/TEST0: PASS
obj_pool/TEST0: SETUP (check/pmem/nondebug)
./obj_pool: Relink `/lib/x86_64-linux-gnu/libkmod.so.2' with `/lib/x86_64-linux-gnu/libpthread.so.0' for IFUNC symbol `system'
obj_pool/TEST0: PASS
obj_pool/TEST0: SETUP (check/pmem/static-debug)
./obj_pool.static-debug: Relink `/lib/x86_64-linux-gnu/libkmod.so.2' with `/lib/x86_64-linux-gnu/libpthread.so.0' for IFUNC symbol `system'
obj_pool/TEST0: PASS
obj_pool/TEST0: SETUP (check/pmem/static-nondebug)
./obj_pool.static-nondebug: Relink `/lib/x86_64-linux-gnu/libkmod.so.2' with `/lib/x86_64-linux-gnu/libpthread.so.0' for IFUNC symbol `system'
obj_pool/TEST0: PASS
obj_pool/TEST0: SETUP (check/non-pmem/debug)
./obj_pool: Relink `/lib/x86_64-linux-gnu/libkmod.so.2' with `/lib/x86_64-linux-gnu/libpthread.so.0' for IFUNC symbol `system'
obj_pool/TEST0: PASS
obj_pool/TEST0: SETUP (check/non-pmem/nondebug)
./obj_pool: Relink `/lib/x86_64-linux-gnu/libkmod.so.2' with `/lib/x86_64-linux-gnu/libpthread.so.0' for IFUNC symbol `system'
obj_pool/TEST0: PASS
obj_pool/TEST0: SETUP (check/non-pmem/static-debug)
./obj_pool.static-debug: Relink `/lib/x86_64-linux-gnu/libkmod.so.2' with `/lib/x86_64-linux-gnu/libpthread.so.0' for IFUNC symbol `system'
obj_pool/TEST0: PASS
obj_pool/TEST0: SETUP (check/non-pmem/static-nondebug)
./obj_pool.static-nondebug: Relink `/lib/x86_64-linux-gnu/libkmod.so.2' with `/lib/x86_64-linux-gnu/libpthread.so.0' for IFUNC symbol `system'
obj_pool/TEST0: PASS

```

**Comment by lplewa** [comment]
https://bugs.launchpad.net/ubuntu/+source/kmod/+bug/1723042

**Comment by panlinux** [comment]
The bug you reported mentions a udeb, not a normal deb. Was that correct? In what env are you using a udeb? FWIW I reported there that trying your test case with a normal bionic installation worked just fine.

**Comment by lplewa** [comment]
Fixed in Ubuntu 18.04, please verify the issue (PMDK compiled with NDCTL_ENABLE=y).

"Ubuntu 17.10 is going to be EOL soon (july 2018, see https://wiki.ubuntu.com/)"
